### PR TITLE
v1.9 backports 2020-11-19

### PR DESCRIPTION
**Pull request by kaworu**
* #13936 -- bpf: don't override DROP_FRAG_NOT_FOUND error (@jibi)
 * #13347 -- bpf: add metrics for fragmented ipv4 packets (@jibi)
 * #14061 -- docs: Fix helm install command in kubeadm getting started guide (@pchaigno)
 * #14064 -- docs: Fix wording around labels configuration (@joestringer)
 * #14029 -- helm/hubble-relay: fixed indentation error (@PranaviRoy)
 * #14012 -- fqdn: Fix confusion of ToFQDNs vs. DNS rules. (@jrajahalme)
 * #14019 -- helm: 'upgradeCompatibility' needs to be a string, not a float64 (@mvisonneau)
 * #13660 -- docs: encryption: interface clarifications (@kkourt)
 * #14030 -- Mention MTU kernel bug in frag page (@liuyuan10)
 * #13961 -- Revert commits that skip running tests on CI with EKS (@christarazi)
 * #13364 -- contrib: Add script to bump stable docker image tags (@joestringer)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 13936 13347 14061 14064 14029 14012 14019 13660 14030 13961 13364; do contrib/backporting/set-labels.py $pr done 1.9; done
```

### CODE REVIEW

**Review by jibi — APPROVED** [review]
My changes and external contributors' ones LGTM :+1: thanks!

**Review by pchaigno — APPROVED** [review]
:+1: for my PR.

**Review by ti-mo — APPROVED** [review]

**Comment by kAworu** [comment]
test-backport-1.9

**Review by christarazi — APPROVED** [review]
LGTM for my commits

**Review by joestringer — APPROVED** [review]

**Comment by kAworu** [comment]
GitHub action failure is related to https://github.com/cilium/cilium/pull/13347 (see https://github.com/cilium/cilium/runs/1395336745) which can be safely ignored if I understand correctly.

/cc @pchaigno @jibi

**Comment by pchaigno** [comment]
> GitHub action failure is related to #13347 (see https://github.com/cilium/cilium/runs/1395336745) which can be safely ignored if I understand correctly.

That's correct. Checkpatch checks if new files were added or removed and notifies us in that case as we might need to update the `CODEOWNERS` file. It however isn't smart enough to figure out whether the new files are already covered by `CODEOWNERS` or not.

@qmonnet Should we disable that warning? All our `bpf/` code is covered by `CODEOWNERS` so maybe not too useful after all?

**Comment by qmonnet** [comment]
> @qmonnet Should we disable that warning? All our `bpf/` code is covered by `CODEOWNERS` so maybe not too useful after all?

I was considering it already, it's more often annoying than useful. Ok le'ts do it, I'll prepare a PR to update the image later today.

File: pkg/fqdn/dnsproxy/proxy_test.go [hunk 1]
```diff
@@ -661,6 +661,27 @@ func (s *DNSProxyTestSuite) TestFullPathDependence(c *C) {
...
+		}}.Sort(),
+		54: restore.IPRules{{
+			Re: restore.RuleRegex{Regexp: s.proxy.allowed[epID1][54][cachedWildcardSelector]},
+		}},
+	}
+	restored1b := s.proxy.GetRules(uint16(epID1)).Sort()
+	c.Assert(restored1b, checker.DeepEquals, expected1b)
+
+	// unlimited again
+	s.proxy.usedServers = nil
```
Comment (jrajahalme) [inline pkg/fqdn/dnsproxy/proxy_test.go:683]: This can cause test flakes, fixed by #14085, which also needs a backport.

**Review by jrajahalme — APPROVED** [review]
Looks good for my part, one more backport needed for unit test stability, though.

File: pkg/fqdn/dnsproxy/proxy_test.go (hunk 1 above)
Comment (joestringer) [reply pkg/fqdn/dnsproxy/proxy_test.go:683]: Next round of backports should pick this up so I'm fine to merge as-is.

**Pull request merged**